### PR TITLE
[ACR] Helm version validation added

### PR DESCRIPTION
**Pull request by hkuadithya**
Issue https://github.com/Azure/acr/issues/243

**New changes have been tested and corresponding screenshots have been added below.**

**Default behavior**

**New changes - helm version < 2.11.0**

**New changes - helm version regex pattern changed; default output is shown**

**Note**: 
1. Azure-cli codebase was refactored. History file has not been modified.
2. Pylint ignore comment has been added to prevent merge failures.

@ni-reis 

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR has modified HISTORY.rst describing any customer-facing, functional changes. Note that this does not include changes only to help content. (see [Modifying change log](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log)).

- [x] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

### CODE REVIEW

File: src/azure-cli/azure/cli/command_modules/acr/check_health.py [hunk 1]
```diff
@@ -136,6 +139,7 @@ def _get_cli_version():
 # Get helm versions
 def _get_helm_version(ignore_errors):
     from ._errors import HELM_VERSION_ERROR
+    from distutils.version import LooseVersion  # pylint: disable=import-error,no-name-in-module
```
Comment (ni-reis) [inline src/azure-cli/azure/cli/command_modules/acr/check_health.py]: Is it safe to disable "import-error"? Won't it break any customers using older versions of python?

File: src/azure-cli/azure/cli/command_modules/acr/check_health.py [hunk 2]
```diff
@@ -153,7 +157,20 @@ def _get_helm_version(ignore_errors):
...
+
+    # Retrieve the helm version if regex pattern is found
+    match_obj = HELM_VERSION_REGEX.search(output)
+    if match_obj:
+        output = match_obj.group(1)
+
+    print("Helm version: {}".format(output))
+
+    # Display an error message if the current helm version < min required version
+    if match_obj and LooseVersion(output) < LooseVersion(MIN_HELM_VERSION):
```
Comment (ni-reis) [inline src/azure-cli/azure/cli/command_modules/acr/check_health.py]: What's the difference between `LooseVersion` and `StrictVersion` ?

File: src/azure-cli/azure/cli/command_modules/acr/check_health.py (hunk 1 above)
Comment (hkuadithya) [reply src/azure-cli/azure/cli/command_modules/acr/check_health.py]: Looks like it's safe. It's been used in a few other places in the CLI codebase.

For instance:
https://github.com/Azure/azure-cli/blob/dev/src/azure-cli-core/azure/cli/core/util.py#L139

https://github.com/Azure/azure-cli/blob/dev/src/azure-cli/azure/cli/command_modules/vm/_actions.py#L110

File: src/azure-cli/azure/cli/command_modules/acr/check_health.py (hunk 2 above)
Comment (hkuadithya) [reply src/azure-cli/azure/cli/command_modules/acr/check_health.py]: From what I've been able to understand, `StrictVersion` raises exception if the software versioning is not appropriate or doesn't follow standard format. Strings such as a993.#@1.332 would raise exceptions.

If the versioning system follows industry standards, i.e.  majorVersion.minorVersion.Patch, `StrictVersion` and `LooseVersion` will have similar behavior.

In the CLI codebase, `LooseVersion` has been used throughout for comparing software versions. I didn't find any uses of `StrictVersion`. Therefore, I felt that `LooseVersion` would be more appropriate for our use case.

Comment (ni-reis) [reply src/azure-cli/azure/cli/command_modules/acr/check_health.py]: I see, makes sense. Good to know. Thanks!

File: src/azure-cli/azure/cli/command_modules/acr/check_health.py (hunk 1 above)
Comment (ni-reis) [reply src/azure-cli/azure/cli/command_modules/acr/check_health.py]: Good to know it's been used in other places in CLI code itself. Should be safe then. Thanks!

**Comment by ni-reis** [comment]
LGTM. @sajayantony can you please follow up with this PR? Thanks!

**Pull request merged**